docs/rcm-api.yaml

```diff
@@ -0,0 +1,91 @@
+definitions:
```

Comment:
I think we should not commit this. I don't see the advantage in having this in the repo.

Reply:
Agreed.